doc/overview.rst

```diff
@@ -8,22 +8,27 @@ Installation
 Native installation
 +++++++++++++++++++
 
-For now, you will need to clone LISA from `github
-<https://github.com/ARM-software/lisa>`_ and use ``setup.py`` to install the
-required packages. The simplest way to do so would be to issue these commands::
+For now, you will need to clone LISA from `github <https://github.com/ARM-software/lisa>`_ ,
+and then issue these commands::
 
   git clone https://github.com/ARM-software/lisa.git -b next
   cd lisa
-  python3 -m pip install -e .
-
-Extra (optionnal) packages can also be installed, for instance notebook support::
-
-  python3 -m pip install -e .[notebook]
+  # A few packages need to be installed, like python3 or kernelshark. Python
+  # modules will be installed in a venv at the next step, without touching 
+  # any system-wide install location.
+  ./install_base_ubuntu.sh
+  # On the first run, it will take care of creating a Python venv and populate it
```

Comment:
```suggestion
  # On the first run, it will take care of creating a Python venv and populating it
```
Or
```suggestion
  # On the first run, it will take care of creating a Python venv and will populate it
```